src/com/parse4cn1/operation/IncrementFieldOperation.java

```diff
@@ -48,7 +48,6 @@ public Object apply(Object oldValue, ParseObject parseObject, String key)
             throws ParseException {
 
         if (oldValue == null) {
-            needIncrement = false;
```

Comment:
Isn't it a programmer's error to be attempting to increment a field that does not have an initial value? From that point of view, I disagree with this change as we're adjusting the library to cater for a programmer's error... Or do you see it differently?